src/PostgREST/PgQuery.hs

```diff
@@ -241,7 +241,7 @@ pgFmtJsonbPath :: T.Text -> T.Text
 pgFmtJsonbPath p =
   pgFmtJsonbPath' $ fromMaybe (ColIdentifier p) (parseJsonbPath p)
   where
-    pgFmtJsonbPath' (ColIdentifier i) = pgFmtIdent i
+    pgFmtJsonbPath' (ColIdentifier i) = i
```

Comment:
This makes me worried that we're opening a place for sql injection. Is the identifier sure to be already escaped before getting into this function?